campus/package.json

```diff
@@ -9,7 +9,7 @@
     "analyze": "webpack --mode=production --env analyze",
     "lint": "eslint src --ext js,ts,tsx",
     "format": "prettier --write \"./**\"",
-    "test": "cross-env BABEL_ENV=test jest",
+    "test": "cross-env BABEL_ENV=test jest --passWithNoTests",
```

Comment:
1. What if someone writes a test and want to run it locally? -> He might find the test passes anyway.
2. What if later we have the test phase in CI? -> All these args need to be removed.

Reply:
> 1. What if someone writes a test and want to run it locally? -> He might find the test passes anyway.
> 2. What if later we have the test phase in CI? -> All these args need to be removed.

@Allen931 Please DO NOT use --passWithNoTests